test/conftest.py

```diff
@@ -316,13 +315,13 @@ def pudl_sql_io_manager(
     # All the hard work here is being done by the datapkg and
     # datapkg_to_sqlite fixtures, above.
     context = build_init_resource_context()
-    return pudl_sqlite_io_manager(context)
+    return pudl_io_manager(context)
 
 
 @pytest.fixture(scope="session")
-def pudl_engine(pudl_sql_io_manager: PudlSQLiteIOManager) -> sa.Engine:
+def pudl_engine(pudl_mixed_format_io_manager: PudlMixedFormatIOManager) -> sa.Engine:
     """Get PUDL SQL engine from io manager."""
-    return pudl_sql_io_manager.engine
+    return pudl_mixed_format_io_manager._sqlite_io_manager.engine
```

Comment:
I didn't add a getter because this is the only place we directly access the `_sqlite_io_manager` like this, and this fixture doesn't make sense if the SQLite manager doesn't exist, so it felt fine in this testing context to make that assumption.